### PR TITLE
adding the ptable help to manual

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by mmoll** [comment]
@shiramax please rebase

**Comment by shiramax** [comment]
@mmoll Done.

File: _includes/manuals/1.16/4.4.4_prov_ptables.md [hunk 1]
```diff
@@ -120,3 +120,50 @@ cat <<EOF > /tmp/diskpart.cfg
...
+clearpart --all --initlabel
+part swap --size 250 --maxsize "$swap_size" --grow
+part /boot --fstype ext3 --size 100 --asprimary
+part / --fstype ext3 --size 8192 --maxsize 12288 --grow
+part /tmp2 --size 250 --fstype ext3 --grow
+EOF
+```
+The inclusion of the keyword string #Dynamic at the start of a line lets Foreman know that this is not an explicit 
+disk layout and must treated as a shell script, executed prior to the install process and that the explicit partition 
+table will be found at /tmp/diskpart.cfg during the build process.
```
Comment (mmoll) [inline _includes/manuals/1.16/4.4.4_prov_ptables.md]: I'd enclose `#Dynamic` and `/tmp/diskpart.cfg` in that sentence with backticks, 👍 otherwise

**Review by mmoll — CHANGES_REQUESTED** [review]

**Comment by shiramax** [comment]
@mmoll, 
Done, Please let me know if this is what you meant.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
this is fine, merged, thanks @shiramax!

**Pull request merged**